src/sparseml/optim/manager.py

```diff
@@ -118,19 +123,76 @@ def compose_staged(
         if not isinstance(additional_recipe, BaseManager):
             additional_recipe = cls.from_yaml(additional_recipe)
 
-        if isinstance(base_recipe.modifiers, OrderedDict):
-            raise ValueError(
-                "non-staged recipes not yet supported for Manager.compose_staged "
-                "found base_recipe with non_staged modifiers"
+        # Both base_recipe and additional_recipe are non-staged_recipes
+        if isinstance(base_recipe.modifiers, List) and isinstance(
+            additional_recipe.modifiers, List
+        ):
+            # Need to generate stage names for two standard recipes
+            base_stage_name, additional_stage_name = [f"stage_{i}" for i in range(2)]
+            base_stages = {base_stage_name: deepcopy(base_recipe.modifiers)}
+
+            additional_stages = {
+                additional_stage_name: deepcopy(additional_recipe.modifiers)
+            }
+
+            base_recipe_metadata = deepcopy(base_recipe._metadata)
+            additional_recipe_metadata = deepcopy(additional_recipe._metadata)
+
+            base_recipe_metadata[base_stage_name] = base_recipe_metadata.pop(
+                "__metadata__"
             )
-        if isinstance(additional_recipe.modifiers, OrderedDict):
-            raise ValueError(
-                "non-staged recipes not yet supported for Manager.compose_staged "
-                "found additional_recipe with non_staged modifiers"
+            additional_recipe_metadata[
+                additional_stage_name
+            ] = additional_recipe_metadata.pop("__metadata__")
```

Comment:
why do we need to pop the "__metadata__" key? why does an unstaged recipe have this key?

Reply:
So the way how I think about `manager._metadata` is:
if `manager`'s recipe is staged:
manager._metadata = {stage_name1: <metadata1>, stage_name2: <metadata2>, ...}
if `manager`'s recipe is not staged:
manager._metadata = {__metadata__: <metadata>}

this construction ```dict[new_key] = dict.pop(old_key)``` is one liner for replacing `old_key` with `new_key`.

So when working with e.g staged checkpoint recipe and new standard recipe:
1. `checkpoint_manager._metadata =  {stage_name1: <metadata1>, stage_name2: <metadata2>, ...}`
2.  compute `stage_name3`
2. `new_manager._metadata =  {stage_name3: <metadata3>}` 
3.  Combine -> combined_manager._metadata =  {stage_name1: <metadata1>, stage_name2: <metadata2> stage_name3: <metadata3>}`

Does this make sense?